Documentation/HowToUseWCFInContainers.md

```diff
@@ -0,0 +1,142 @@
+This document describes how to get started with WCF for .NET Core in containers.
+
```

Comment:
Can we add instructions of prerequisite, for example, what OS, what need to be installed and what are the links?